test/suite/test_rollback_to_stable05.py

```diff
@@ -133,12 +133,11 @@ def test_rollback_to_stable(self):
         self.assertEqual(calls, 1)
         self.assertEqual(keys_removed, 0)
         self.assertEqual(keys_restored, 0)
+        self.assertGreaterEqual(pages_visited, 0)
```

Comment:
@raviprakashgiri29 Can you add a new test case to demonstrate this inconsistent problem. It is possible that eviction can go ahead and write data that is ahead of the checkpoint.

Reply:
Added a new test case test_checkpoint_snapshot02.py and new stats counter to demonstrate the inconsistent checkpoint scenario in which eviction writes more than data than a checkpoint.